server/service/src/backend_plugin/boajs/methods/sql.rs

```diff
@@ -19,7 +20,9 @@ pub(crate) fn bind_method(context: &mut Context) -> Result<(), JsError> {
                 let connection = service_provider
                     .connection()
                     .map_err(std_error_to_js_error)?;
-                raw_query(&connection, sql).map_err(std_error_to_js_error)
+                raw_query(&connection, sql.clone())
+                    .inspect_err(|e| log::error!("{} {sql}", format_error(e)))
```

Comment:
Quite a lot of sql error while developing, wanted to print the whole sql at this point